nextcloud-server.yaml

```diff
@@ -0,0 +1,401 @@
+#nolint:valid-pipeline-git-checkout-tag
+package:
+  name: nextcloud-server
+  version: 31.0.5
+  epoch: 0
+  description: "Nextcloud server, a safe home for all your data"
+  copyright:
+    - license: AGPL-3.0-or-later
+  dependencies:
+    runtime:
+      - coreutils
+      - findutils
+      - grep
+      - sed
+      - flock
+      - redis
+      - rsync
+      - su-exec
+      - diffutils
+      - apache2-utils
+      - apache2-compat
+      - procps
+      - apache2
+      - bash
+      - dash-binsh
+      - docker-library-php
+      - docker-library-php-compat
+      - busybox
+      - bzip2
+      - libldap
+      - libmagic
+      - imagemagick
+      - openssl
+      - php-${{vars.php-version}}
+      - php-${{vars.php-version}}-config
+      - php-${{vars.php-version}}-apache
+      - php-${{vars.php-version}}-bcmath
+      - php-${{vars.php-version}}-exif
+      - php-${{vars.php-version}}-ftp
+      - php-${{vars.php-version}}-fpm
+      - php-${{vars.php-version}}-gd
+      - php-${{vars.php-version}}-gmp
+      - php-${{vars.php-version}}-intl
+      - php-${{vars.php-version}}-apcu
+      - php-${{vars.php-version}}-ldap
+      - php-${{vars.php-version}}-igbinary
+      - php-${{vars.php-version}}-imagick
+      - php-${{vars.php-version}}-memcached
+      - php-${{vars.php-version}}-sodium
+      - php-${{vars.php-version}}-opcache
+      - php-${{vars.php-version}}-pcntl
+      - php-${{vars.php-version}}-pdo
+      - php-${{vars.php-version}}-pdo_mysql
+      - php-${{vars.php-version}}-mysqlnd
+      - php-${{vars.php-version}}-pdo_pgsql
+      - php-${{vars.php-version}}-sysvsem
+      - php-${{vars.php-version}}-zip
+      - php-${{vars.php-version}}-redis
+      - php-${{vars.php-version}}-pdo_sqlite
+      - php-${{vars.php-version}}-iconv
+      - php-${{vars.php-version}}-ctype
+      - php-${{vars.php-version}}-dom
+      - php-${{vars.php-version}}-curl
+      - php-${{vars.php-version}}-xml
+      - php-${{vars.php-version}}-simplexml
+      - php-${{vars.php-version}}-fileinfo
+      - php-${{vars.php-version}}-mbstring
+      - php-${{vars.php-version}}-curl
+      - php-${{vars.php-version}}-openssl
+      - php-${{vars.php-version}}-posix
+      - php-${{vars.php-version}}-xmlreader
+      - php-${{vars.php-version}}-xmlwriter
+      - rsync
+      - ${{package.name}}-apache2-config
+  scriptlets:
+    post-install: |
+      #!/bin/sh
+      # As wolfi-baselayout does provide /var/spool/cron already, and we can not create this
+      # directory in the package, we need to create the cron file in the post-install scriptlet.
+      # Since scriptlets don't run in apko, this is for the `apk add` command only.
+      mkdir -p /var/spool/cron/crontabs || true
+      echo '*/5 * * * * php -f /var/www/html/cron.php' > /var/spool/cron/crontabs/www-data
+
+vars:
+  php-version: 8.1
```

Comment:
Their docs note that PHP 8.1 is deprecated: Can this be built with with 8.3 as recommended? https://docs.nextcloud.com/server/latest/admin_manual/installation/system_requirements.html

Reply:
Nice catch! https://github.com/wolfi-dev/os/pull/51763/commits/608fc0a7877c3d1728eb9a6c7cf5c3ef23a4fbfe